cuda_name: cuda:0
running on  GEN_davis
Pre-processed data found: data\processed\davis_train.pt, loading ...
Pre-processed data found: data\processed\davis_test.pt, loading ...
Training on 20036 samples...
Train epoch: 1 [0/20036 (0%)]	Loss: 29.572983
C:\Python311\Lib\site-packages\torch_geometric\deprecation.py:26: UserWarning: 'data.DataLoader' is deprecated, use 'loader.DataLoader' instead
  warnings.warn(out)
Train epoch: 1 [162840/20036 (25%)]	Loss: 1.115464
Train epoch: 1 [324120/20036 (51%)]	Loss: 0.826907
Train epoch: 1 [495480/20036 (76%)]	Loss: 0.814465
Make prediction for 5010 samples...
running on  HGC_GCN_davis
Pre-processed data found: data\processed\davis_train.pt, loading ...
Pre-processed data found: data\processed\davis_test.pt, loading ...
Traceback (most recent call last):
  File "C:\Users\dshre\Documents\Sem-6\BTP\DeepNC\training_validation.py", line 95, in <module>
    train(model, device, train_loader, optimizer, epoch + 1)
  File "C:\Users\dshre\Documents\Sem-6\BTP\DeepNC\training_validation.py", line 32, in train
    output = model(data)
             ^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\dshre\Documents\Sem-6\BTP\DeepNC\models\hgc_gcn.py", line 39, in forward
    x = self.conv1(x, edge_index)
        ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch\nn\modules\module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch\nn\modules\module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch_geometric\experimental.py", line 117, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\torch_geometric\nn\conv\hypergraph_conv.py", line 167, in forward
    assert hyperedge_attr is not None
AssertionError
Training on 20036 samples...